spec/lib/tasks/destroy_unused_providers_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+require './lib/tasks/destroy_unused_providers'
+
+RSpec.describe DestroyUnusedProviders::DestroyableRecords do
+  let(:iu) { create(:integration_usage) }
+  let(:iaa_order) { iu.iaa_order }
+  let(:integration) { iu.integration }
+  let(:service_provider) { integration.service_provider }
+  let(:in_person_enrollment) {  create(:in_person_enrollment, service_provider: service_provider) }
+
+  subject(:destroyable_records) { described_class.new(service_provider.issuer) }
+
+  before do
+    # silencing output
+    allow_any_instance_of(DestroyUnusedProviders::DestroyableRecords).to receive(:puts)
```

Comment:
Is this still needed?

Reply:
ah, no. i can either get rid of it or update it silencing Rails.logger.debug -- do we have a best practice about silencing output in tests?

Reply:
I don't remember what our default log device, or log level is for rake tasks have we confirmed this will go to STDOUT in prod?

We could:
```ruby
Rails.logger = Logger.new(STDOUT, level: :debug)
```
somewhere in this file to make sure it's logged as expected? and update specs to give it a null logger (`Logger.new('/dev/null')`) to minimize noise when running specs